analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -171,7 +171,9 @@ class GoMod(
 
         val list = run("list", "-m", "all", workingDir = projectDir)
         list.stdout.lines().forEach { line ->
-            val columns = line.split(' ')
+            // For replaced modules the line is formatted as "${orig-module} => ${replaced-module}", see also
+            // https://go.dev/ref/mod#go-mod-file-replace.
+            val columns = line.substringAfterLast(" => ").split(' ')
```

Comment:
While at it, you could make this more compact, similar to what I suggested in your other PR:
```kotlin
return list.stdout.lineSequence()
    .map { it.substringAfterLast(" => ").split(' ') }
    .filter { it.size == 2 }
    .map { Identifier(...) }
```
But I'll approve anyway as it's unrelated to the change itself.